kong/plugins/rate-limiting/handler.lua

```diff
@@ -29,6 +29,17 @@ local function get_identifier(conf)
     end
   elseif conf.limit_by == "credential" then
     identifier = ngx.ctx.authenticated_credential and ngx.ctx.authenticated_credential.id
+  elseif conf.limit_by == "key" then
+    identifier = kong.request.get_header(conf.key_name) or kong.request.get_query_arg(conf.key_name)
+
+    if not identifier and conf.key_in_body then
+      local body, err = kong.request.get_body()
+      if err then
+        ngx_log(ngx.ERR, "cannot process request body: ", tostring(err))
```

Comment:
Could you use `kong.log.err` here?
https://docs.konghq.com/0.14.x/pdk/kong.log/#kong-log-level